Packs/PrivX/Integrations/PrivX/PrivX.yml

```diff
@@ -0,0 +1,145 @@
+category: Identity and Access Management
+commonfields:
+  id: PrivX
+  version: -1
+configuration:
+- defaultvalue: 
+  display: Your PrivX server address (FQDN or IP address)
+  name: hostname
+  required: true
+  type: 0
+- defaultvalue: 443
+  display: Your PrivX server HTTPS port
+  name: port
+  required: true
+  type: 0
+- defaultvalue: privx-external
+  display: OAuth client ID
+  name: oauth-client-id
+  required: true
+  type: 0
```

Comment:
All the authentication params need to be of type 9 - [credentials](https://xsoar.pan.dev/docs/integrations/code-conventions#credentials)